.github/workflows/ci.yaml

```diff
@@ -15,12 +15,27 @@ jobs:
       - name: Set up Go 1.x
         uses: actions/setup-go@v2
         with:
-          go-version: ^1.17
+          go-version: 1.17
         id: go
 
       - name: Checkout code into the Go module directory
         uses: actions/checkout@v2
 
+      - name: Tidy
+        run: |
+          rm -f go.sum
+          go mod tidy -go=1.16 && go mod tidy -go=1.17
```

Comment:
I don't see any advantage of having this in CI. As devs, we should tidy the go.sum before opening PRs so that CI and outside users don't need to be bothered of it.

Reply:
I see. I'll remove this step.

Reply:
It would have been nice if we could detect a situation that is not tidy in advance and cause the github actions to fail. However, since this step is just to run the `go mod tidy`, not to fail, we decided to find another better way.